.github/workflows/apply-final-merge-readiness.yml

```
@@ -0,0 +1,644 @@
+
+on:
+  push:
+    branches:
+      - v2.6.5
+
+permissions:
+  contents: write
+
+jobs:
+  apply:
+    if: github.event.head_commit.message == 'chore: stage final merge-readiness patch'
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)